databricks/koalas/tests/test_frame_spark.py

```diff
@@ -92,3 +92,32 @@ def test_repartition(self):
         new_kdf = kdf.spark.repartition(num_partitions)
         self.assertEqual(new_kdf.to_spark().rdd.getNumPartitions(), num_partitions)
         self.assert_eq(kdf.sort_index(), new_kdf.sort_index())
+
+    def test_coalesce(self):
+        num_partitions = 10
+        kdf = ks.DataFrame({"age": [5, 5, 2, 2], "name": ["Bob", "Bob", "Alice", "Alice"]})
+        kdf = kdf.spark.repartition(num_partitions)
+
+        new_kdf = kdf.spark.coalesce(--num_partitions)
```

Comment:
Hmm... I don't think maybe Python doesn't support such decrement operator ??

```python
>>> num_partitions = 10
>>> --num_partitions
10
```

Maybe I think you should specify '9' if you want give '9' as a parameter.

Reply:
Thanks for catching this! Modified.